### PR TITLE
Feature events section

### PR DESCRIPTION
**Pull request by keptup**

### CODE REVIEW

File: src/components/Events/Events.js [hunk 1]
```diff
@@ -1,9 +1,68 @@
...
+                alt={event.title}
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                {event.title}
+              </a>
+            </span>
+            <span>{event.date}</span>
+            <a
+              href={event.links["Google Maps"]}
```
Comment (spukas) [inline src/components/Events/Events.js]: You are using quite a lot of `event.`, it is a good sign to destruct variables `id, title, links...` before the return statement.

File: src/components/Events/Events.js [hunk 2]
```diff
@@ -1,9 +1,68 @@
...
+    </ul>
+  );
+
+  return (
+    <React.Fragment>
+      <div className="events-container">
+        <h1>ARTIMIAUSI KONCERTAI</h1>
+        {gigs.length < 1 ? noEvents : upcomingEvents}
+      </div>
+    </React.Fragment>
```
Comment (spukas) [inline src/components/Events/Events.js]: Fragment is used for cases where elements are not wrapped in to the parent component. Now you have a `React.Fragment` as a parent component and `<div className="events-container">` as a parent component. So Fragment is not needed here.

File: src/components/Events/Events.js [hunk 3]
```diff
@@ -1,9 +1,68 @@
...
+  //Filter future gigs from past ones
+  const gigs = eventData.filter(event => {
+    const [year, month, day] = event.date.split("-");
+    // converting strings to numbers with '+' instead of Number();
+    const eventDate = new Date(+year, +month - 1, +day + 1);
+    return eventDate > new Date();
+  });
+
+  const noEvents = (
+    <p className="noGigsMessage">
```
Comment (spukas) [inline src/components/Events/Events.js]: The classname does not match the pattern of the BEM methodology.

File: src/components/Events/Events.css [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+@media only screen and (min-width: 961px) {
+  .events-container {
+    min-height: 500px;
+    max-width: 800px;
+  }
+}
+
+.events-container h1 {
+  text-align: center;
+  margin: 35px auto 25px auto;
```
Comment (spukas) [inline src/components/Events/Events.css]: Just for the future to think about: would be nice to have a separate file, like `spacings.css` with predefined spacing variables like `--small-spacing; --big-spacing...` etc. These variables could be imported into the necessary files, thus making sure we would have a consistent spacings over the app :)

File: src/components/Events/Events.js (hunk 1 above)
Comment (keptup) [reply src/components/Events/Events.js]: Fixed

File: src/components/Events/Events.js (hunk 2 above)
Comment (keptup) [reply src/components/Events/Events.js]: Fixed

File: src/components/Events/Events.js (hunk 3 above)
Comment (keptup) [reply src/components/Events/Events.js]: Fixed

File: src/components/Events/Events.css (hunk 4 above)
Comment (keptup) [reply src/components/Events/Events.css]: I could use some help undestanding this deeper.

File: src/components/Events/Events.js [hunk 5]
```diff
@@ -1,9 +1,68 @@
...
 function Events() {
-    return (
-        <h2>This is and Event section</h2>
-    )
-};
+  //Filter future gigs from past ones
+  const gigs = eventData.filter(event => {
+    const [year, month, day] = event.date.split("-");
+    // converting strings to numbers with '+' instead of Number();
+    const eventDate = new Date(+year, +month - 1, +day + 1);
```
[the remark this answers is not in the document]
Comment (keptup) [reply src/components/Events/Events.js]: If I get it right, adding 0:00:00 to the end of the new Date declaration is needed to get it work properly.
`new Date('2019-05-23 0:00:00')`
I believe adding these zeros to every declaration of the new Date() would be even more problematic than it is now.

[the remark this answers is not in the document]
Comment (spukas) [reply src/components/Events/Events.js]: It works without zeros. You can try it by writing in the browser console:
`new Date('2019-05-23') < new Date()` => true

[the remark this answers is not in the document]
Comment (keptup) [reply src/components/Events/Events.js]: It works in part, but somehow it does not display gig if it were to occur today (`<=` not working in this case). Still sticking to the old solution.

[the remark this answers is not in the document]
Comment (spukas) [reply src/components/Events/Events.js]: True, ok then.

**Pull request merged**